### ISSUE TITLE
KaldiDecoder	Missing HCLG.fst	Graph not found at .../graph/HCLG.fst.

### ISSUE THREAD

**Issue by basnijholt** [synesthesiam/rhasspy-hassio-addon#181]
I am using a HassIO installed Rhasspy 2.4.18 on a RPi 4.

I am getting the error consistently:
> KaldiDecoder | Missing HCLG.fst | Graph not found at /share/rhasspy/profiles/en/kaldi/model/graph/HCLG.fst. Did you train your profile?

I have removed my entire `/share/rhasspy` and reinstalled the add-on, however, the error returns. All of my relevant config files are [here](https://github.com/basnijholt/home-assistant-config/tree/master/rhasspy).

I have found [this](https://community.home-assistant.io/t/rhasspy-offline-voice-assistant-toolkit/60862/891?u=basnijholt) where @synesthesiam suggested [this](https://community.home-assistant.io/t/rhasspy-offline-voice-assistant-toolkit/60862/895?u=basnijholt), however, that didn't solve it for me.

Another person [reported](https://community.rhasspy.org/t/number-range-not-working/398/9?u=basnijholt) the same problem. There, `tobetobe` removed the `kaldi` folder and the problem was solved. This also didn't help in my case.

[This person also reported](https://community.rhasspy.org/t/stt-speeds-on-local-pi-setup/300/42?u=basnijholt) the same problem. There "deleting the settings" is suggested, however, it's unclear to me which settings should be removed.

Several others seem to have the problem, judging from [this search](https://community.rhasspy.org/search?q=Missing%20HCLG.fst).

**Comment by synesthesiam** [comment]
There must be an error during training then, since the HCLG.fst is not being produced. This could also be caused if you have "open transcription" checked in your settings.

**Comment by basnijholt** [comment]
Thanks for your reply, `"Open transcription mode (no custom voice commands)"` is turned off.

Considering many others experience this problem, I am not sure whether it's due to my sentences.

Would you know how to debug this?

**Comment by mathquis** [comment]
I often had this error when Rhasspy had to generate pronunciations for unknown words when using Kaldi.
What I did to fix it is download the profile files again and re-train.
If it doesn't work, try looking at your `kaldi/custom_words.txt` file to check if everything is ok (does it have a new line at the end, etc.).
Hope this helps.

**Comment by basnijholt** [comment]
@mathquis, indeed, thanks a lot. The culprit was my `custom_words.txt` (see below). I removed them and now it works again.
```
kef K EH F
spotify S P AA T IH F AY
```

@synesthesiam, shouldn't there be a better error message that points people in the right direction>?

**Comment by synesthesiam** [comment]
The custom words look fine to me. Was there anything in the log from Kaldi about them?
